spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
             profile::core::docker
```

Comment:
Replace with `include_examples 'docker', docker_version: '24.0.9'`

Reply:
i couldn't use the shared_examples for docker because the hexrot uses `storage_driver: 'devicemapper'`
but i included a test for this condition.